File: datacenter-proxies/Java/src/main/java/ConsoleWriter.java
public class ConsoleWriter {
    public void writelnAndExit(String str) {
        this.writelnError(str);
        System.exit(1);
    }

    public void writelnError(String output) {
        this.writeln("ERROR: " + output);
    }

    public void writeln(String str) {
        System.out.println(str);
    }
}

File: web-unblocker/CSharp/Oxylabs/ConsoleWriter.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Oxylabs
{
    class ConsoleWriter
    {
        public void WritelnAndExit(String str)
        {
            this.WritelnError(str);
            System.Environment.Exit(1);
        }

        public void WritelnError(String output)
        {
            this.Writeln("ERROR: " + output);
        }

        public void Writeln(String str)
        {
            Console.WriteLine(str);
        }
    }
}
